runtimes/animation-playback.mdx

```diff
@@ -335,133 +337,13 @@ export default function App() {
   <Tab title="Flutter">
     Flutter handles things a little differently compared to the other runtimes due to its reactive nature.
 
-    Every animation and state machine in Flutter has an underlying controller that manages the state of each animation. When you pass a list of animation names to the RiveAnimation widget, it creates and manages controllers for each.
-
-    In order to access controls for animations, you'll need to instantiate a RiveAnimationController for each animation and pass the controller to the RiveAnimation widget instead of its name. You can mix and match passing in controllers and names, but avoid passing in both for the same animation.
-
-    There are a number of controllers provided in the runtime that perform certain tasks. We'll cover these as they become relevant.
-
-    #### Manually controlling a looping animation
-
-
-    [SimpleAnimation](https://pub.dev/documentation/rive/latest/rive/SimpleAnimation-class.html) is a basic controller that provides simple playback control of a single animation. With this controller, you can play, pause, and reset animations.
-
-    In the following example, the `isActive` property of `SimpleAnimation` is used to play and pause a single animation.
+    Every animation and state machine in Flutter has an underlying painter that manages the state/painting/advancing.
```

Comment:
There is no code sample for this under the `Flutter` tab. Should there be?

Reply:
I wanted to avoid full blown code examples. This is shown in the example app though, and we can point to that in the future. But I also want to (once the Rive flutter package is updated) make life code samples, similar to browserstack.

But the other reason for not showing it is because this whole section is deprecated.